apis/python/src/tiledbsoma/dataframe.py

```diff
@@ -41,13 +41,13 @@ def __init__(
         self._cached_is_sparse = None
 
     @property
-    def soma_type(self) -> Literal["SOMADataFrame"]:
-        return "SOMADataFrame"
+    def soma_type(self) -> Literal["DataFrame"]:
```

Comment:
The abstract spec says that this returns "SOMADataFrame", which makes sense as that string is not the Python class, but the SOMA API "type".  I think you want all `soma_type` functions to be left as is.

CC: @thetorpedodog

Reply:
This is what I was thinking too; that the `soma_type` property would remain `SOMAWhatever`.